.devops/deploy-pipelines.yml

```
@@ -318,42 +318,150 @@ stages:
   # --- END Deploy UAT --- #
 
   # --- START Deploy PROD --- #
-  - stage: "Prod_WaitForApproval"
-    displayName: 'PROD approval deployment'
-    dependsOn: [ Deploy_UAT_Green,Release ]
-    condition: |
-      and(
-        eq(${{parameters.UAT_PROD_DEPLOY}}, true),
-        eq(dependencies.Deploy_UAT_Green.result, 'Succeeded'),
-        in(dependencies.Release.result, 'Succeeded', 'Skipped')
-      )
-    variables:
-      release_url: $[ stageDependencies.Release.make_release.outputs['chart_current_version.releaseUrl'] ]
-    jobs:
-      - job: Prod_Approval
-        displayName: Manual prod deploy approval
-        pool: server
-        timeoutInMinutes: 4320 # 3 days
-        steps:
-          - task: ManualValidation@0
-            timeoutInMinutes: 4320 # 3 days
-            inputs:
-              notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
-              instructions: "Please approve or reject PROD promotions for release $(release_url)"
-              onTimeout: 'reject'
+  # - stage: "Prod_WaitForApproval"
+  #   displayName: 'PROD approval deployment'
+  #   dependsOn: [ Deploy_UAT_Green,Release ]
+  #   condition: |
+  #     and(
+  #       eq(${{parameters.UAT_PROD_DEPLOY}}, true),
+  #       eq(dependencies.Deploy_UAT_Green.result, 'Succeeded'),
+  #       in(dependencies.Release.result, 'Succeeded', 'Skipped')
+  #     )
+  #   variables:
+  #     release_url: $[ stageDependencies.Release.make_release.outputs['chart_current_version.releaseUrl'] ]
+  #   jobs:
+  #     - job: Prod_Approval
+  #       displayName: Manual prod deploy approval
+  #       pool: server
+  #       timeoutInMinutes: 4320 # 3 days
+  #       steps:
+  #         - task: ManualValidation@0
+  #           timeoutInMinutes: 4320 # 3 days
+  #           inputs:
+  #             notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
+  #             instructions: "Please approve or reject PROD promotions for release $(release_url)"
+  #             onTimeout: 'reject'
```

```suggestion
  - stage: "Prod_WaitForApproval"
    displayName: 'PROD approval deployment'
    dependsOn: [ Deploy_UAT_Green,Release ]
    condition: |
      and(
        eq(${{parameters.UAT_PROD_DEPLOY}}, true),
        eq(dependencies.Deploy_UAT_Green.result, 'Succeeded'),
        in(dependencies.Release.result, 'Succeeded', 'Skipped')
      )
    variables:
      release_url: $[ stageDependencies.Release.make_release.outputs['chart_current_version.releaseUrl'] ]
    jobs:
      - job: Prod_Approval
        displayName: Manual prod deploy approval
        pool: server
        timeoutInMinutes: 4320 # 3 days
        steps:
          - task: ManualValidation@0
            timeoutInMinutes: 4320 # 3 days
            inputs:
              notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
              instructions: "Please approve or reject PROD promotions for release $(release_url)"
              onTimeout: 'reject'
```

deploy in PROD env should be manually approved before start deployment stage